src/Framework/ref/Microsoft.AspNetCore.App.Ref.csproj

```diff
@@ -36,10 +36,15 @@ This package is an internal implementation of the .NET Core SDK and is not meant
 
     <!-- Reference implementation assemblies in addition to ref assemblies to get xml docs -->
     <ReferenceImplementationAssemblies>true</ReferenceImplementationAssemblies>
+    <ResolveAssemblyReferencesSilent>true</ResolveAssemblyReferencesSilent>
 
     <!-- Platform manifest data -->
     <FrameworkListFileName>FrameworkList.xml</FrameworkListFileName>
     <FrameworkListOutputPath>$(ArtifactsObjDir)$(FrameworkListFileName)</FrameworkListOutputPath>
+
+    <!-- Platform manifest and package override metatdata -->
+    <ReferencePackSharedFxVersion>$(AspNetCoreMajorVersion).$(AspNetCoreMinorVersion).0</ReferencePackSharedFxVersion>
```

Comment:
Won't this mean preview builds generate a Microsoft.AspNetCore.App.Ref package without the version suffix? Suggest adding `-$(VersionSuffix)` when that is non-empty.